packages/core/src/parameters/getParameterDefinitionsFromSource.js

```diff
@@ -0,0 +1,174 @@
+const JSON5 = require('json5')
+
+const countSpaces = (l) => {
```

Comment:
In general, there should be a few comments in the code to help developers understand what's happening. Of course, the originator understands NOW, but how about in 3-5 years?

Note: These are not comments for JSDOC